lib/websocket.go

```diff
@@ -20,7 +44,16 @@ type WebsocketHandler interface {
 // and serves to maintain the list of connections
 type wsHandler struct {
 	// Collect all websocket connections
-	conns []*websocket.Conn
+	conns         map[string]*wsConn
+	connsLock     sync.Mutex
+	keystore      key.Store
+	subscriptions map[string]string
+	subLock       sync.Mutex
+}
+
+type wsConn struct {
+	profileID string
+	conn      *websocket.Conn
```

Comment:
terrible name

Not sure what to call this. It's here to help us associate profileIDs and with connIDs in the "other" direction. maybe just calling it a Conn is okay? or would that be too confusing with the other kinds of conns?

Reply:
all of these `ws` prefixes are a clue this should be moved into a sub package `github.com/qri-io/qri/lib/websocket`. Rename `WebsocketHandler` to `websocket.Handler`, and keep only the interface and constructor exported.

Also, this struct will need an `ID` field in the end so we can transition back to "anonomyous" connections